android-exoplayer/build.gradle

```diff
@@ -61,6 +61,8 @@ dependencies {
 
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
 
+    implementation "com.google.protobuf:protobuf-javalite:3.14.0"
```

Comment:
What is protobuf required for?

Reply:
That's for key id extraction from stream, which introduced from exoDoris 0.15.1.

Reply:
We use protobuf to parse the pssh private data, to get the keyId from stream.
For mobile video player it use firebase and have the dependence of protobuf-javalite:3.14.0, but the TV video player hasn't this dependence, and I add it explicitly for TV video player.